### PR TITLE
Update the README file

### LINKED ISSUE #14: Update the readme file
Update the README file with all relevant information of the project

### PR DESCRIPTION
**Pull request by jeniaSakirko**
This PR is for updating the readme file with basic information about the project
You can see that the contributing file from #2 is mentioned so the PR should be merged after the file

closes #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,53 @@
...
+Project Hotails is an open-source platform for dog daycares management. Hotails main goal is to provide both for dogs
+owners and dog daycares managers, an efficient, comfortable and reliable system.
+
+## Getting Started
+
+These instructions will get you up and running on your local machine.
+
+### Prerequisites
+
+What things you need to install the software and how to install them:
```
Comment (Omeramsc) [inline README.md:18]: maybe `in order to install the software you should.....`

File: README.md [hunk 2]
```diff
@@ -1,9 +1,53 @@
...
+
+A step by step series that tell you how to get Hotails up and running
+
+1. Clone the repository
+2. Open command prompt in Hotails directory
+3. Spin the environment using `vagrant up` 
+4. That's it! your virtual machine is up and running 
+5. Go to http://localhost:8000 in your web browser
+6. You can enter the virtual machine by running `vagrant ssh`
+7. When finished, run `vagrant destroy -f` to tear down the environment
```
Comment (Omeramsc) [inline README.md:33]: Maybe add a note about all changes getting deleted after doing so.

**Review by Omeramsc — CHANGES_REQUESTED** [review]
Added a few comments

BTW, you could've removed the `contributing.md` mention and have the contributing PR add it, therefore removing the dependencies between them, :)

**Comment by jeniaSakirko** [comment]
> Added a few comments
> 
> BTW, you could've removed the `contributing.md` mention and have the contributing PR add it, therefore removing the dependencies between them, :)

What do you think about commenting out the contributing part and uncomment in new PR?

File: README.md (hunk 2 above)
Comment (jeniaSakirko) [reply README.md]: Added

**Review by OfirMatasas — APPROVED** [review]

**Review by ErezCohenn — COMMENTED** [review]
optional suggestion:
1) adding architecture section that will look something like that:
# Architecture

| Frontend      | Backennd      | Datebase      |
| ------------- | ------------- | ------------- |
| HTML          | Python        | SQLite        |
| CSS           | Django        |  
|  Bootstarp      | Vagrant       |

2) adding team members section that will include our names and photos (optional)

**Comment by Omeramsc** [comment]
> > Added a few comments
> > BTW, you could've removed the `contributing.md` mention and have the contributing PR add it, therefore removing the dependencies between them, :)
> 
> @Omeramsc What do you think about commenting out the contributing part and uncomment in new PR?

Think about how a new user or developer will interpret it. Comments can be misleading (especially when on a markdown file, and not in code), it is better to either remove it altogether (and so, remove the dependencies), or leave it as is.

**Comment by jeniaSakirko** [comment]
> optional suggestion:
> 
> 1. adding architecture section that will look something like that:
> 
> # Architecture
> Frontend	Backennd	Datebase
> HTML	Python	SQLite
> CSS	Django	
> Bootstarp	Vagrant	
> 2. adding team members section that will include our names and photos (optional)

1. I think adding the architecture will be relevant in the future now we will only have one column
2. Added the team members

**Review by ErezCohenn — APPROVED** [review]
LGTM

**Review by Yuval-Vino — APPROVED** [review]
After your changes, this README file looks real good! 
Great job!!!

**Review by Omeramsc — APPROVED** [review]
LGTM

**Pull request merged**